File: WebBS.Data/Models/Item.cs
using System;

namespace WebBS.Data.Models
{
    public class Item
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Name2 { get; set; }
        public double CostPrice { get; set; }
        public double Price { get; set; }
        public double Coverage { get; set; }
        public string Supplier { get; set; }
        public Guid Category { get; set; }
    }
}

File: FrontEnd_ReactJs/ClientApp/src/components/Items/Item.js
import React from "react";

function Item(props){
console.log(props)
    return(
        <div>
            <h5>{props.object.name}</h5>
            <p>Vendor {props.object.name2}</p>
            <p>id: {props.object.id}</p>
        </div>
    )
}

export default Item